includes/WP_Admin.php

```diff
@@ -24,10 +25,23 @@ final class WP_Admin {
 	 * @return void
 	 */
 	public function __construct() {
+		\add_action( 'init', array( __CLASS__, 'load_php_textdomain' ) );
 		\add_action( 'admin_menu', array( __CLASS__, 'register_page' ) );
 		\add_action( 'load-dashboard_page_' . self::$slug, array( __CLASS__, 'initialize' ) );
 	}
 
+	/**
+	 * Loads the textdomain for the module. This applies only to PHP strings.
+	 *
+	 * @return boolean
+	 */
+	public static function load_php_textdomain() {
+		return I18nService::load_php_translations(
+			'wp-module-onboarding',
```

Comment:
Don't think so, this is the only file where the string `wp-module-onboarding` can be used as a variable. `__` mandates the text domain argument to be a string literal (since those need to be parsed before runtime) and thus cannot be constants/variables. Don't see the point of creating a variable just for this file.

Reply:
For Reference: https://markjaquith.wordpress.com/2011/10/06/translating-wordpress-plugins-and-themes-dont-get-clever/

Reply:
LGTM!